pym/bob/scm/git.py

```diff
@@ -174,12 +174,22 @@ def getProperties(self, isJenkins, pretty=False):
             properties.update({GitScm.REMOTE_PREFIX+key : val})
         return properties
 
+    def _getGitConfigOptions(self):
+        config = [ "-c", "submodule.recurse=0" ]
+        if self.__url.startswith("file:") or self.__url.startswith("/"):
+            config += [ "-c", "protocol.file.allow=always" ]
+        return config
+
     async def invoke(self, invoker, switch=False):
         alternatesFile = invoker.joinPath(self.__dir, ".git/objects/info/alternates")
 
         # make sure the git directory exists
         if not os.path.isdir(invoker.joinPath(self.__dir, ".git")):
             await invoker.checkCommand(["git", "init", self.__dir])
+            if self.__url.startswith("file:"):
+                await invoker.checkCommand(["git", "config", "--local",
```

Comment:
This looks odd. First, isn't the `self.__url.startswith("/")` test missing? Then, why is `_getGitConfigOptions` needed if the option is already set for the repository?

Reply:
Oh...This was the first attempt to get the tests back working on my system. Sorry - I overlooked it during cleanup. :see_no_evil: